iputils_srcs/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+rm -rf iputils
+
+# echo "Downloading the source code"
+git clone https://github.com/iputils/iputils iputils
+
+# echo "Configuring..."
+pushd iputils &> /dev/null || exit 1
+./configure && meson build
+
+# echo "Installing the build..."
+pushd builddir &> /dev/null || exit 1
+meson install
+
+# echo "Copying ping, tracepath, clockdiff, arping..."
+mkdir -p ../../bin
+cp ./arping ./clockdiff ./tracepath ./ping/ping ../../bin/
+
+popd &> /dev/null
+popd &> /dev/null
+../extract.sh ./bin/tracepath
```

Comment:
```suggestion

../extract.sh ./bin/tracepath
```

Reply:
Yeah, my bad `xsltproc` should have been listed as a dependency. I modified the README to include it as well. It should work now.